recipe/meta.yaml

```diff
@@ -20,14 +20,12 @@ requirements:
     - python                                # [build_platform != target_platform]
     - cython                                # [build_platform != target_platform]
     - meson-python                          # [build_platform != target_platform]
-    - ninja                                 # [build_platform != target_platform]
-    - scikit-learn >=1.5.0,<1.6             # [build_platform != target_platform]
+    - ninja
     - {{ stdlib('c') }}
     - {{ compiler('c') }}
   host:
     - python
     - meson-python
-    - ninja
```

Comment:
ninja is an executable, meant to be run by the "building" machine at compile time only. it should only be in the build section.